spec/operators/throttleTime-spec.ts

```diff
@@ -55,12 +55,48 @@ describe('Observable.prototype.throttleTime', () => {
     expectSubscriptions(e1.subscriptions).toBe(subs);
   });
 
-  it('should handle a busy producer emitting a regular repeating sequence', () => {
-    const e1 =   hot('abcdefabcdefabcdefabcdefa|');
-    const subs =     '^                        !';
-    const expected = 'a-----a-----a-----a-----a|';
+  it('should handle a busy producer emitting a regular repeating sequence with leading: true, trailing: false', () => {
+    const e1 =   hot('a12345b12345c12345d123|');
+    const subs =     '^                     !';
+    const expected = 'a-----b-----c-----d---|';
 
-    expectObservable(e1.throttleTime(50, rxTestScheduler)).toBe(expected);
+    expectObservable(e1.throttleTime(50, rxTestScheduler, {leading: true, trailing: false})).toBe(expected);
```

Comment:
if you could use the `time` function here and represent the `time` as marbles above like:

```
const t1 = '----|';
```

and then use `time(t1)` which returns a number.

That would be ideal as the test could be more readable, you can even used commented out `----|` marblse to demonstrate where the throttles occur.

Remember, the whole purpose of these tests is to be more readable.

Reply:
Please do the same for the other tests.